modules/compliance-manual.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-manual_{context}"]
+= Remediating a platform check manually
+
+Checks for Platform scans typically have to be remediated manually by the administrator for two reasons:
+
+. It is not always possible to guess the value that must be set. One of the checks requires that a list of allowed registries is provided, but the scanner has no way of knowing which registries the organization wants to allow.
+. Different checks modify different API objects.
```

Comment:
Great point.  How about:
```Because different checks modify different API objects, automated remediation would require `root` or superuser access to modify objects in the cluster, which is not advised. ```